docs/tutorial/force_calibration.rst

```diff
@@ -104,3 +118,89 @@ We can plot the calibration by calling::
+These equations can be used to fit a hydrodynamically correct power spectrum:
+
+.. math::
+
+    P_{hydro}(f) = \frac{D Re(\gamma / \gamma_0)}{\pi^2 \left(\left(f_{c,0} + f Im(\gamma/\gamma_0) - f^2/f_{m, 0}\right)^2 + \left(f Re(\gamma / \gamma_0)\right)^2\right)}
+
+where
+
+.. math::
+
+    f_{c, 0} = \frac{\kappa}{2 \pi \gamma_0}
+
+and
+
+.. math::
+
+    f_{m, 0} = \frac{\gamma_0}{2 \pi m}
+
+with :math:`m` the mass of the bead and :math:`\gamma` the frequency dependent drag coefficient.
+Assuming a measurement deep in bulk, these are given by:
+
+.. math::
+
+    \gamma_{stokes} = \gamma_0 \left(1 + (1 - i)\sqrt{\frac{f}{f_{\nu}}} - \frac{2}{9}\frac{f}{f_{\nu}} i\right)
+
+Where :math:`f_{\nu}` is given by :math:`4 \nu/(\pi d^2)` with :math:`\nu` the kinematic viscosity.
+
+This approximation is reasonable, when the bead is far from the surface.
+When approaching the surface, the drag experienced by the bead depends on the distance between the center of the bead and the nearest surface of the flow cell.
+An approximate expression for the frequency dependent drag coefficient is then given by :cite:`tolic2006calibration`:
+
+.. math::
+
+    \gamma(f, R/l) = \frac{\gamma_{stokes}(f)}{1 - \frac{9}{16}\frac{R}{l}\left(1 - \left((1 - i)/3\right)\sqrt{\frac{f}{f_{\nu}}} + \frac{2}{9}\frac{f}{f_{\nu}}i - \frac{4}{3}(1 - e^{-(1-i)(2l-R)/\delta})\right)}
+
+where :math:`\delta = R \sqrt{\frac{f_{\nu}}{f}}` and :math:`R` corresponds to the bead radius, :math:`l` to the distance from the bead center to the nearest surface.
+
+While these models may look daunting, they are all available in Pylake and can be used by simply providing a few additional arguments to the :class:`~.PassiveCalibrationModel`.
+It is recommended to use these equations when less than 10% systematic error is desired :cite:`tolic2006calibration`.
+
+The figure below shows the difference between the hydrodynamically correct spectrum and the regular Lorentzian for various bead sizes.
+
+.. image:: hydro.png
+
+These more advanced models require a few extra parameters namely the density of the sample, density of the bead and distance to the surface (in meters)::
+
+    force_model = lk.PassiveCalibrationModel(bead_diameter, hydrodynamically_correct=True, rho_sample=999, rho_bead=1060.0, distance_to_surface=1e-6)
+
+Note that when `rho_sample` and `rho_bead` are omitted, values for water and polystyrene are used for the sample and bead density respectively.
+
+Additionally, when the parameter `distance_to_surface` is omitted, a simpler model is used which assumes the experiment was performed deep in bulk (neglecting the increased drag induced by the nearby surface).
+
+Fast Sensors
+------------
+
+Fast detectors provide a higher bandwidth, meaning that they do not need the effect of the diode in the model.
+
+We can omit the diode response model by passing `fast_sensor=True` to the `CalibrationModel`::
```

Comment:
When is a sensor fast? Can you give a relation between the bandwidth of the sensor and eg the corner frequency in the power spectrum to say what is fast?

Reply:
I've rephrased it as such:
```
Fast detectors respond much faster resulting in no visible filtering effect by the detector in the frequency range we are fitting, meaning that they do not need to model the slower response time of the diode in the model.
```
Most users will likely not have a fast sensor.

In the future, the hope is to read this from Bluelake and avoid users from having to set this flag themselves.